config/studio/studio_version.xml

```diff
@@ -14,7 +14,6 @@
   ~ You should have received a copy of the GNU General Public License
   ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   -->
-
 <studio>
-    <version>3.1.4.4</version>
+    <version>3.1.9</version>
```

Comment:
Is this the minimum version?

Reply:
Reverted for compatibility with other versions